### PR TITLE
Helm chart fixes

### PR DESCRIPTION
**Pull request by sdreyer**

### CODE REVIEW

File: charts/featureform/charts/api-server/values.yaml [hunk 1]
```diff
@@ -43,7 +43,7 @@ service:
   port: 7878
 
 apiserver:
-  port: 7878
+  port: "7878"
```
Comment (Sami1309) [inline charts/featureform/charts/api-server/values.yaml:46]: why r there quotes here and not around the other 7878

Comment (sdreyer) [reply charts/featureform/charts/api-server/values.yaml:46]: leftover from how I was doing it before. Will fix

Comment (sdreyer) [reply charts/featureform/charts/api-server/values.yaml]: Fixed

**Review by Sami1309 — APPROVED** [review]
lgtm

**Pull request merged**